README.md

```diff
@@ -100,10 +100,27 @@ Config options overview:
 | password | smtp    |                             | Password for email client.                                    |
 | host     | smtp    | smtp.gmail.com              | Host of the email client.                                     |
 
+### Transformation configuration
+
+Configuration files for TranSMART and cBioPortal must be placed in `transformation_config_dir`. 
+Specifically, the following are expected:
+
+- `sources_config.json` and `ontology_config.json`, described in [python_csr2transmart](https://github.com/thehyve/python_csr2transmart#usage); 
+the files reference the input data and need to be customized accordingly,
+- `portal.properties` file for cBioPortal; 
+the file  must match the mounted cBioPortal image version and server environment, 
+- `cbioportal_db_info` folder, containing configuration files for the cBioPortal database 
+(`cancertypes.json`, `genes.json`, `genesaliases.json`); 
+the files must match the mounted cBioPortal image version.
+ 
+Sample configuration files are provided in the repository under `test_data/test_logic_NGS/config`.
```

Comment:
I don't see a directory like this, should it be `test_data/test_data_NGS/config`? Also, this can be a link to the folder: [test_data/test_data_NGS/config](https://github.com/thehyve/pmc-conversion/tree/master/test_data/test_data_NGS/config).

Reply:
Thanks for spotting it, I updated it with a link!